domain-model/src/test/java/uk/gov/hmcts/cmc/domain/utils/ResponseUtilsTest.java

```diff
@@ -139,4 +139,23 @@ public void shouldReturnFalseWhenNonAdmissionResponse() {
         Response response = FullDefenceResponse.builder().build();
         assertThat(ResponseUtils.isAdmissionResponse(response)).isFalse();
     }
+
+    @Test
+    public void shouldReturnFalseWhenFullAdmissionResponseForIsPartAdmission() {
```

Comment:
Can you rename test cases and make it simpler e.g. shouldReturnFalseWhenFullAdmissionResponse and you don't need to mention method name I think.